packages/functions/src/primitive-outline.ts

```diff
@@ -0,0 +1,135 @@
+import { type Document, Primitive, Triangle, type vec3 } from '@gltf-transform/core';
+import { vec3 as glvec3 } from 'gl-matrix';
+import { assignDefaults, createTransform } from './utils.js';
+
+const NAME = 'primitive-outline';
+
+export interface PrimitiveOutlineOptions {
+	thresholdAngel: number;
+}
+
+export const PRIMITIVE_OUTLINE_DEFAULTS: Required<PrimitiveOutlineOptions> = {
+	thresholdAngel: 0.05,
+};
+
+export function createPrimitiveOutline(_option: PrimitiveOutlineOptions = PRIMITIVE_OUTLINE_DEFAULTS): Primitive {
```

Comment:
Currently the convention in the repository is for transforms (functions passed to `document.transform(...)`) to have very concise names, and I'm undecided about how to expose this as a transform, I think it would require some ways to target particular meshes.

I'd prefer, at least for now, to remove the createPrimitiveOutline transform from this PR, and instead to export the `createEdgePrimitive` function as a standalone function taking a Primitive as input, like:

```typescript
export function createEdgePrimitive(prim: Primitive, thresholdRadians: number): Primitive {
  const graph = prim.getGraph();
  const document = Document.fromGraph(graph)!;
  ...
}
```

... does that sound OK?